src/Traits/Tag.php

```diff
@@ -234,12 +236,15 @@ public function close()
      */
     public function render()
     {
+        $htmlData = $this->open().$this->getContent().$this->close();
```

Comment:
This should be initialized in the `tidy` elseif as it's not needed in the other cases